---
layout: page
title: Data
---

The genome of the mediterranean sea urchin *Paracentrotus lividus* was assembled using a combination of Illumina data provided by [Genoscope](http://www.genoscope.cns.fr) and Pacbio and HiC data collected by members of the Paracentrotus lividus genome consortium. After primary assembly, haplotypes were reconcilied using haplomerger.

This assembly spans 927.48Mb (scaf N50: 41.5Mb) split in 3,747 scaffolds with 5% of residual gaps. Masked regions using a custom repeat library  make up 44.3% of the genomes and are represented as lower case characters (soft-masking); residual gaps in the assembly are represented with Ns.

A draft annotation of the genome was generated by training Augustus using transcript aligned with PASA and subsquently polishing these ab initio models using EVM incoporating transcript alignment and protein alignments from seastar and amphioxus. UTRs and splice variants were added using 2 rounds of refinement against the PASA transcript aligment. This annotation contains 30593 genes (probably an overestimate) of which 23573 have blast similarity against Strongylocentrotus genes (if so, 'gene_name' field was filled based on SPU name in the GTF).
Other ressources generated are indicated in the table below:

|File   |Dataset   | Format |
|---|---|---|
| **[Pliv_PqN3S_sm.fa](https://www.dropbox.com/s/uf9q1o7op5psjzi/Pliv_PqN3S_sm.fa.gz?dl=1)**  | Softmasked genome assembly  | fasta |
| **[Pliv_PqN3S_evmp.gtf](https://www.dropbox.com/s/4b0f11ooj2fwa1a/Pliv_PqN3S_evmp.gtf.gz?dl=1)** | EVM/PASA annotation | GTF |
| **[Pliv_PqN3S_evmp.pep](https://www.dropbox.com/s/yn3v1zhoxqo8q7y/Pliv_PqN3S_evmp.pep.gz?dl=1)** | EVM/PASA proteins | fasta |
| **[Pliv_PqN3S_stageStt.gtf](https://www.dropbox.com/s/op02lw4p1pneq19/Pliv_PqN3S_stageStt.gtf.gz?dl=1)** | Stringtie RNA-seq | GTF |
| **[Pliv_PqN3S_stageStt.gtf](https://www.dropbox.com/s/9kimik7jsj1n6th/Pliv_aH2p_fg.gtf?dl=1)** | New Annotation | GTF |
| **[Pliv_aH2p_fg.gtf](https://www.dropbox.com/s/9kimik7jsj1n6th/Pliv_aH2p_fg.gtf?dl=1)** | New Annotation | GTF |
| **[annot_equivalence.tsv](https://www.dropbox.com/s/grukyhrtigw75b2/annot_equivalence.tsv?dl=1)** | gene and transcript equivalence between annotation | GTF |

The new annotation uses improved RNA-seq and contains more protein domains than the previous one, however, it is still a work in progress! 


## Genome browser

*Please note that a new server for the hub has been deployed, because the previous instance didn't have enouh memory for BLAT to run!*

The sea urchin genome assembly and annotation are available as a [UCSC track hub](https://genome.ucsc.edu/goldenPath/help/hgTrackHubHelp.html). The assembly should be searchable using BLAT. To enable access, you can either:

- Use the [following UCSC session](http://genome-asia.ucsc.edu/s/fmarletaz/hub_80016_Pliv_PqN3S)
- set up the hub by adding the hub address `http://95.179.158.79/Urchin/hub.txt` to `My Data > Track Hubs > My hubs` in UCSC menu (see [instructions as pdf](https://www.dropbox.com/s/oqk7e1kh8ch7ezq/hub.pdf?dl=0)).


